packages/apollo-mock-server/package.json

```diff
@@ -24,15 +24,10 @@
     "cross-fetch": "^3.0.4",
     "express": "^4.17.1",
     "graphql": "^15.0.0",
+    "graphql-tag": "^2.11.0",
     "hops-config": "^14.0.0-nightly.8",
     "hops-mixin": "^14.0.0-nightly.8",
     "hops-webpack": "^14.0.0-nightly.8"
   },
-  "peerDependencies": {
```

Comment:
In theory, removing the peer dependencies is invalid, because we still need them (**if** we are using Hops v14 with Apollo v2).
But we don't want new users to install Hops v14 and then use Apollo v2, therefore I think it is okay to remove these peer dependencies. Because what we actually want with this PR is: Old apps that still use Apollo v2 should continue to work and new apps should use Apollo v3 anyway.
WDYT?